### PR TITLE
postgresql_privs: fix schema quoting

### LINKED ISSUE #381: postgresql_privs: schema name is not quoted properly
### bug in postgresql_privs

When trying to grant privileges on something in a schema, the schema is not quoted properly, which results in an error.

When the `schema` argument contains no special quoting, the postgres raises an syntax error.
When quotes are included in the `schema` argument, the module aborts because the check if the schema exists fails. (`Error: Schema ""c.d"" does not exist.`)

This is tested with community.postgresql 2.3.1 and postgresql 10-15.

#### steps to reproduce

```yaml
tasks:
  - name: create db
    community.postgresql.postgresql_db:
      name: "a.b"
      login_unix_socket: /tmp
  - name: create schema
    community.postgresql.postgresql_schema:
      database: "a.b"
      name: '"c.d"'
      login_unix_socket: /tmp
  - name: create table
    community.postgresql.postgresql_table:
      db: "a.b"
      name: '"c.d"."e.f"'
      columns:  []
      login_unix_socket: /tmp
  - name: grant something on the database
    community.postgresql.postgresql_privs:
      login_unix_socket: /tmp
      database: "a.b"
      privs: CREATE
      roles: PUBLIC
      type: database
  - name: grant privileges on a schema
    community.postgresql.postgresql_privs:
      login_unix_socket: /tmp
      database: "a.b"
      privs: CREATE
      roles: PUBLIC
      type: schema
      objs: 'c.d'
  - name: grant privileges on a table in a schema
    community.postgresql.postgresql_privs:
      login_unix_socket: /tmp
      database: "a.b"
      privs: SELECT
      roles: PUBLIC
      type: table
      objs: ALL_IN_SCHEMA
      schema: "{{ item }}"
    loop:
      - 'c.d'
      - '"c.d"'
    ignore_errors: true # it fails
```

### PR DESCRIPTION
**Pull request by betanummeric**
fixes #381

Make postgresql_privs work with schema names containing arbitrary characters (e.g. `"` or `.`).

### CODE REVIEW

File: plugins/modules/postgresql_privs.py [hunk 1]
```diff
@@ -817,6 +817,7 @@ def manipulate_privs(self, obj_type, privs, objs, orig_objs, roles, target_roles
         if not objs:
             return False
 
+        quoted_schema_qualifier = '"%s"' % schema_qualifier.replace('"', '""') if schema_qualifier else None
```
[the remark this answers is not in the document]
Comment (betanummeric) [reply plugins/modules/postgresql_privs.py:820]: https://www.postgresql.org/docs/current/sql-syntax-lexical.html documents:
> To include the escape character in the identifier literally, write it twice.

So to have a literal `"` inside the quoted identifier, we need to double it.

[the remark this answers is not in the document]
Comment (betanummeric) [reply plugins/modules/postgresql_privs.py:820]: I will extend the test later.

[the remark this answers is not in the document]
Comment (Andersson007) [reply plugins/modules/postgresql_privs.py:820]: OK, SGTM, thanks for clarifying

**Review by Andersson007 — APPROVED** [review]

**Comment by betanummeric** [comment]
@Andersson007 It appears to be applicable to version 1 too, I will create a PR for that.

**Pull request merged**